flytekit/tools/repo.py

```diff
@@ -59,12 +63,22 @@ def serialize_get_control_plane_entities(
     :param settings: SerializationSettings to be used
     :param pkgs: Dot-delimited Python packages/subpackages to look into for serialization.
     :param local_source_root: Where to start looking for the code.
+    :param is_registration: Whether this is happening for registration.
     """
     settings.source_root = local_source_root
     ctx_builder = FlyteContextManager.current_context().with_serialization_settings(settings)
     with FlyteContextManager.with_context(ctx_builder) as ctx:
         registrable_entities = get_registrable_entities(ctx, options=options)
-        click.secho(f"Successfully serialized {len(registrable_entities)} flyte objects", fg="green")
+        if is_registration:
```

Comment:
Since `is_registration` is only for printing, can this printing logic be moved up to the caller?

Reply:
sure! good call